packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/SplitButton.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.forms;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import java.text.MessageFormat;
+
+import org.eclipse.sirius.components.annotations.Immutable;
+import org.eclipse.sirius.components.forms.validation.Diagnostic;
+
+/**
+ * The Splitbutton widget.
+ * It contains a list of Button widget.
+ *
+ * @author mcharfadi
+ */
+@Immutable
+public final class SplitButton extends AbstractWidget {
+
+    private String label;
+
+    private java.util.List<Button> actions = new ArrayList<>();
```

Comment:
Why using qualified `java.util.List` here?

Reply:
Because we have a java class named `List `in the same package here.

Reply:
It doesn't matter, you already use `java.utilList` in this file so you just have to continue.